src/OrchardCore.Modules/OrchardCore.Taxonomies/Liquid/TaxonomyTermFilter.cs

```diff
@@ -38,6 +51,7 @@ public async ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArgument
             {
                 taxonomyContentItemId = jobj["TaxonomyContentItemId"].Value<string>();
                 termContentItemIds = ((JArray)jobj["TermContentItemIds"]).Values<string>().ToArray();
+                localizedCulture = jobj.Root["LocalizationPart"]["Culture"].Value<string>();
```

Comment:
There should be a LocalizedAspect for that. This way you don't depend on a specific part, and it's strongly typed.

Reply:
Not sure if there is, I couldn't get it. Any hint, appreciated..

Reply:
> There should be a LocalizedAspect for that.

I think Seb means `CultureAspect`